[docs] Clarifying renderAppWithRedux props requirements (#3125)
commercetools/merchant-center-application-kit: website/src/content/api-reference/commercetools-frontend-application-shell.mdx

<Info>

In addition to the following options, the method accepts all options from `renderApp`. 

> Please note that you can only pass one of store, storeState, or sdkMocks.

Reviewer: ```suggestion
Please note that you can only pass one of `store`, `storeState`, or `sdkMocks`.
```

> Please note that you can only pass one of `store`, `storeState`, or `sdkMocks`.

Reviewer 2: ```suggestion
Please note that it is not possible to pass either `storeState` or `sdkMocks` together with `store`.
```

Author: Yeah thanks, it was hard to figure out if it was only 1 of each or there was another combination that was possible.
